src/components/templates/SimpleAdvancedView/SimpleAdvancedView.tsx

```diff
@@ -21,9 +24,26 @@ export const SimpleAdvancedView = ({
 
     const component = view === 'Simple' ? simpleComponent : advanceComponent;
 
+    const { isConnected } = useAccount();
+    const balance = useBalances();
+
+    const isShowWelcomeAlert =
+        Object.values(balance).every(v => v === 0) || !isConnected;
```

Comment:
The alert only shows if

1) wallet isn't connected
2) wallet balance is 0 for every currency

Reply:
2. walletBalance is 0 or collateralBalance is 0?

Reply:
Wallet Balance